README.md

```diff
@@ -4,13 +4,13 @@ This is a tiny microbenchmark library for Zig, designed to be easy to use and ve
 
 ## Usage
 
-You can either manually copy `benchmark.zig` into your project, or add it as a dependency using the Zig package manager.
+You can either manually copy `benchmark` into your project, or add it as a dependency using the Zig package manager.
```

Comment:
the `.zig` should stay in this line, though maybe clarifying that it's the `benchmark.zig` *file* that you can copy directly into your project would be good :)

Reply:
updated wording